admin/class-monk-admin.php

```diff
@@ -298,6 +298,30 @@ public function monk_save_post_meta_box( $post_id ) {
 	}
 
 	/**
+	 * Function to filter the query inside the category meta box using the post language
+	 *
+	 * @param    $term_query instance of WP_Term_Query class 
+	 *
+	 * @since    1.0.0
+	*/
+	public function monk_apply_terms_filter( $term_query ) {
+		$screen = get_current_screen();
+		if ( 'edit' === $screen->parent_base ) {
```

Comment:
Add conditional to check if is category query
Add conditional to check `$lang` query var